CONTRIBUTING.md

```diff
@@ -19,15 +19,15 @@
   * [**Action Pack** (controllers, routing) issues](https://github.com/rails/rails/blob/main/guides/bug_report_templates/action_controller_main.rb)
   * [**Generic template** for other issues](https://github.com/rails/rails/blob/main/guides/bug_report_templates/generic_main.rb)
 
-* For more detailed information on submitting a bug report and creating an issue, visit our [reporting guidelines](https://edgeguides.rubyonrails.org/contributing_to_ruby_on_rails.html#reporting-an-issue).
+* For more detailed information on submitting a bug report and creating an issue, visit our [reporting guidelines](https://guides.rubyonrails.org/contributing_to_ruby_on_rails.html#reporting-an-issue).
```

Comment:
Maybe linking the contributing guides to edge still makes sense as these might change before getting a release.